specification/cognitiveservices/data-plane/Language/preview/2023-04-15-preview/analyzeconversations-authoring.json

```
@@ -17,16 +17,10 @@
     "hostTemplate": "{Endpoint}/language",
     "useSchemePrefix": false,
     "parameters": [
-      {
-        "name": "endpoint",
-        "in": "path",
-        "description": "Supported Cognitive Services endpoint (e.g.,\nhttps://<resource-name>.api.cognitiveservices.azure.com).",
-        "required": true,
-        "type": "string"
-      },
       {
         "name": "Endpoint",
         "in": "path",
+        "description": "Supported Cognitive Services endpoint (e.g.,\nhttps://<resource-name>.api.cognitiveservices.azure.com).",
```

```suggestion
        "description": "Supported Cognitive Services endpoint e.g., https://<resource-name>.api.cognitiveservices.azure.com.",
```

Parens unnecessary, and escape characters like "\n" won't work everywhere so avoid any sort of rendering control characters.

---

Thanks for the advice. Will make the changes as suggested here.